Fix broken link in API Playground troubleshooting page (#872)
mintlify/docs: api-playground/troubleshooting.mdx

    requests, then all of these requests will fail. To fix this, configure your reverse proxy to
    allow `POST` requests to the `/api/request` path.

>     Alternatively, if your reverse proxy prevents you from accepting `POST` requests, you can configure Mintlify to send requests directly to your backend with the `api.playground.proxy` setting in the `docs.json`, as described [here](https://mintlify.com/docs/settings#param-proxy). This will

Reviewer: ```suggestion
    Alternatively, if your reverse proxy prevents you from accepting `POST` requests, you can configure Mintlify to send requests directly to your backend with the `api.playground.proxy` setting in the `docs.json`. See [Settings](https://mintlify.com/docs/settings#param-proxy) for more information. This will
```

An option to make the link text more descriptive
